### ISSUE TITLE
cfn-config can no longer be installed due to missing dependency

### ISSUE THREAD

**Issue by crohr** [mapbox/cfn-config#135]
```
$ npm install -g cfn-config
npm ERR! fetch failed https://github.com/mapbox/Inquirer.js/archive/error-handler.tar.gz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://github.com/mapbox/Inquirer.js/archive/error-handler.tar.gz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
[    ..............] - fetchMetadata: WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
```

And indeed https://github.com/mapbox/Inquirer.js no longer exists.

**Comment by crohr** [comment]
Although `npm install -g mapbox/cfn-config` works fine.

**Comment by rclark** [comment]
cc @jakepruitt

**Comment by rclark** [comment]
Hi @crohr! 

We [moved to a namespaced module at v2.5.0](https://github.com/mapbox/cfn-config/blob/master/CHANGELOG.md#250---2-16-12-23). The bug you've got here was fixed in v2.6.1, so that's why the namespaced install worked!
